lib/ansiblelint/constants.py

```diff
@@ -3,3 +3,6 @@
 
 
 DEFAULT_RULESDIR = os.path.join(os.path.dirname(__file__), 'rules')
+
+INVALID_CONFIG_RC = 2
+ANSIBLE_FAILURE_RC = 3
```

Comment:
It'd be better if this was in Enum.

Reply:
I have mixed feelings about that. Lets see how it grows and we can refactor it later.